src/main/java/core/basesyntax/Printer.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Printer {
+    void printFigure();
+}

```

Comment:
`Printer` is abstract separate interface, so method naming should not be tied up to figures.